lambdas/utils/exceptions.py

```diff
@@ -88,19 +88,25 @@ class TagNotFoundException(Exception):
     pass
 
 
-class LogoutFailureException(Exception):
```

Comment:
These exceptions have been moved to `lambda_exceptions.py`

Reply:
Done